### PR TITLE
added image editing with filerobot

### PR DESCRIPTION
**Pull request by Dashrekker**
image

Added image editor to object detection using filerobot. Allows user to implement social media esque image edits.

### CODE REVIEW

File: frontend/playground-frontend/src/components/ObjectDetection/ObjectDetection.js [hunk 1]
```diff
@@ -74,6 +77,19 @@ const ObjectDetection = () => {
     setInputKey((e) => e + 1);
   };
 
+  const dataURLtoFile = (dataurl, filename) => {
+    const arr = dataurl.split(",");
```
Comment (karkir0003) [inline frontend/playground-frontend/src/components/ObjectDetection/ObjectDetection.js:81]: where did you get this code from? @Dashrekker

File: frontend/playground-frontend/src/components/ObjectDetection/ObjectDetection.js [hunk 2]
```diff
@@ -110,7 +120,66 @@ const ObjectDetection = () => {
...
+        }}
+        Text={{ text: "Filerobot..." }}
+        Rotate={{ angle: 90, componentType: "slider" }}
+        Crop={{
+          presetsItems: [
+            {
+              titleKey: "classicTv",
+              descriptionKey: "4:3",
+              ratio: 4 / 3,
+              // icon: CropClassicTv, // optional, CropClassicTv is a React Function component. Possible (React Function component, string or HTML Element)
```
Comment (karkir0003) [inline frontend/playground-frontend/src/components/ObjectDetection/ObjectDetection.js:144]: dead code?

**Review by karkir0003 — CHANGES_REQUESTED** [review]
minor questions/clarifications. overall looking good to me :)

Wait for Faris' comments

**Comment by karkir0003** [comment]
@Dashrekker I conditionally approve your PR. Wait on Faris' comments :)

**Review by karkir0003 — COMMENTED** [review]
address faris' comments

File: frontend/playground-frontend/src/components/general/ImageFileUpload.js [hunk 3]
```diff
@@ -8,6 +8,7 @@ const ImageFileUpload = (props) => {
   const handleFileUpload = (e) => {
     e.preventDefault();
     setUploadFile(e.target.files[0] ? e.target.files[0] : null);
+    console.log(e.target.files[0] ? e.target.files[0] : null);
```
Comment (farisdurrani) [inline frontend/playground-frontend/src/components/general/ImageFileUpload.js:11]: Any reason this console log is here?

Comment (karkir0003) [reply frontend/playground-frontend/src/components/general/ImageFileUpload.js:11]: @Dashrekker

Comment (Dashrekker) [reply frontend/playground-frontend/src/components/general/ImageFileUpload.js:11]: It's for debugging. I can remove it.

Comment (Dashrekker) [reply frontend/playground-frontend/src/components/general/ImageFileUpload.js]: Just removed it.

**Comment by karkir0003** [comment]
@Dashrekker do a `git pull origin dev` followed by a `git push` to make sure this branch is up to date with latest version of `dev`

**Review by karkir0003 — COMMENTED** [review]
update this branch by doing `git pull origin dev`

**Review by karkir0003 — COMMENTED** [review]
do we need the `plain-white-background.jpg` file @Dashrekker

**Review by karkir0003 — APPROVED** [review]
approve as long as the build checks pass

**Review by farisdurrani — APPROVED** [review]

**Pull request merged**